haiku-apps/helios/helios-1.7.2.recipe

```diff
@@ -3,11 +3,11 @@ DESCRIPTION="Cdrecord GUI frontend."
 HOMEPAGE="https://github.com/HaikuArchives/Helios"
 COPYRIGHT="2000-2004 Maurice Michalski"
 LICENSE="MIT"
-REVISION="1"
-SOURCE_URI="git+https://github.com/HaikuArchives/Helios.git#ffbbca7"
-CHECKSUM_SHA256="ce82ede4fd218d3f0c7e30cafc061d2a377e473dbc88512a61e286f21ae51925"
+REVISION="3"
```

Comment:
Your PR hasn't been merged yet, so you don't have to bump revision. Revision bumping is only needed for packages that builds in the main repo

```diff
-REVISION="3"
+REVISION="2"
```

Reply:
Understand. Thanks.